### PR TITLE
Add Paychex to Finance section

### PR DESCRIPTION
**Pull request by ld3300**
Add Paychex to Finance section

### CODE REVIEW

File: _data/finance.yml [hunk 1]
```diff
@@ -59,6 +59,14 @@ websites:
       img: mint.png
       tfa: No
 
+    - name: Paychex
+      url: https://www.paychex.com
+      twitter: paychex  
+      img: paychex.png
+      tfa: Yes
+      sms: Yes
+      doc: https://www.paychex.com/corporate/security/multifactor-authentication.aspx
```
Comment (Carlgo11) [inline _data/finance.yml]: that doc just says that the option will come in a "near future"

File: _data/finance.yml [hunk 2]
```diff
@@ -59,6 +59,14 @@ websites:
       img: mint.png
       tfa: No
 
+    - name: Paychex
+      url: https://www.paychex.com
+      twitter: paychex
```
Comment (Carlgo11) [inline _data/finance.yml]: Twitter handles are not needed when adding a site already using TFA.

**Comment by ld3300** [comment]
They don't appear to have any documentation written after they implemented multi-factor authentication, but I have an account there and have set it up on my account and confirmed it available.  What is the appropriate action to show documentation?

**Comment by Carlgo11** [comment]
Well if you could send a screenshot (with your own private data blurred out) of the TFA login step that would be great.
Then you could just remove the doc link.

And the best thing would be if you contacted Paychex and asked them for a documentation link before removing the doc value.

**Comment by ld3300** [comment]
Thank you.  I have reached out to them and also taken a screenshot.  How would you want the screenshot included?  Thank you for helping me get started with contributing

**Comment by ld3300** [comment]
Here is the screenshot of implementing tfa
paychex screenshot2

**Comment by mxxcon** [comment]
There's no need to add website. Just wanted to confirm that your interpretation of those categories is long the lines with our other entries. :)
Hopefully they'll reply to you soon and we'll merge this.

**Comment by ld3300** [comment]
Thanks again.

**Comment by Carlgo11** [comment]
Okay just remove the `doc` link and we're good to go.

**Pull request merged**